src/test/suite/markdownUtils.test.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+import { expect } from 'chai';
+import * as mdUtils from '../../markdownUtils';
+
+suite('Markdown Utils Test Suite', () => {
+
+	test('get the markdown parser', () => {
+		expect(mdUtils.getMDParser()).to.not.be.null;
```

Comment:
In which case the markdown Parser can be null?

Reply:
it really shouldn't EVER be null. for me this is more a case of "checking a box" for test coverage on what is a utility method.

That said, we will be likely changing the parser for a better GitHub-supported syntax when we investigate https://issues.redhat.com/browse/FUSETOOLS2-637 and I think this is a good placeholder for when that occurs so we can watch for any potential ripples.

Reply:
if it is just "checking a box" for test coverage, it should not be provided. it will give a false impression that the code is tested in a useful way.

Reply:
As I said on https://github.com/redhat-developer/vscode-didact/pull/297, I will be expanding on these tests to make sure that they are more useful. This is just a placeholder ahead of that.